external/harvestPackages/harvestPackages.depproj

```diff
@@ -9,6 +9,11 @@
 
   <Import Project="harvestPackages.props" />
 
+  <ItemGroup>
+    <!-- Only include the assets from the direct packages we reference in the output -->
+    <PackageToInclude Include="@(PackageReference)" />
```

Comment:
We don't binplace in harvestPackages.depproj, is this just for in case?

Reply:
Good point I forgot we don't copy. I will remove this.